gwswlib/hydx.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Hydx:
+    def __init__(self):
+        self.connection_nodes = []
+
+
+class ConnectionNode:
+    MAAIVELDSCHEMATISERINGCOLL = [
```

Comment:
Tenzij er een goede reden is, geen afko zoals "COLL" gebruiken als je "COLLECTION" ofzo bedoelt.
Om aan te sluiten bij de django conventie zou ik het `MAAIVELDSCHEMATISERING_CHOICES` noemen.

Reply:
De COLL komt letterlijk uit de documentation van RioNED, vandaar deze keuze